rmf_traffic_editor/gui/level.cpp

```diff
@@ -542,6 +571,28 @@ bool Level::delete_lift_vertex(std::string lift_name)
   return true;
 }
 
+std::vector<Edge> Level::edges_with_vertex(int vertex_idx) const
+{
+  std::vector<Edge> result;
+  for (const auto& edge : edges)
+  {
+    if (edge.start_idx == vertex_idx || edge.end_idx == vertex_idx)
```

Comment:
I thought you added a `contains` function that can be used here instead?

Reply:
Amended.